temporal/server_impl.go

```diff
@@ -157,7 +158,7 @@ func (s *ServerImpl) Stop() {
 }
 
 // Populates parameters for a service
-func newBootstrapParams(
+func NewBootstrapParams(
```

Comment:
I need to export this and `StopService` methods so that I can reuse them. cc @yiminc